packages/react-ui-core/src/List/List.js

```diff
@@ -0,0 +1,81 @@
+import React, { PureComponent } from 'react'
+import PropTypes from 'prop-types'
+import isEqual from 'lodash/isEqual'
+import cn from 'classnames'
+import { randomId } from '@rentpath/react-ui-utils'
+import themed from 'react-themed'
+import ListItem from './ListItem'
+
+@themed('*', { pure: true })
+
+export default class List extends PureComponent {
+  static propTypes = {
+    theme: PropTypes.object,
+    className: PropTypes.string,
+    nodeType: PropTypes.string,
+    Item: PropTypes.oneOfType([
+      PropTypes.node,
+      PropTypes.string,
```

Comment:
Nitpick, and one I'm not too sure about, but it seems that, according to the docs, `PropTypes.node` includes strings. 
https://reactjs.org/docs/typechecking-with-proptypes.html